### PR TITLE
feat(all contracts): update main to cosmwasm 1.0.0

### PR DESCRIPTION
**Pull request by P-Yevhenii**

### CODE REVIEW

**Review by epanchee — CHANGES_REQUESTED** [review]

File: contracts/tokenomics/xastro_token/src/contract.rs [hunk 1]
```diff
@@ -686,12 +684,15 @@ pub fn query_all_accounts(
...
+        .map(|addr| addr_validate_to_lower(deps.api, &addr))
+        .transpose()?;
+    let start = start.as_ref().map(Bound::exclusive);
 
     let accounts = BALANCES
-        .range(deps.storage, start, None, Order::Ascending)
+        .keys(deps.storage, start, None, Order::Ascending)
         .take(limit)
-        .map(deserialize_pair)
+        .map(|addr| addr.map(String::from))
```
[the remark this answers is not in the document]
Comment (P-Yevhenii) [reply contracts/tokenomics/xastro_token/src/contract.rs:695]: Cool! Thanks

File: packages/astroport/src/math.rs [hunk 2]
```diff
@@ -0,0 +1,12 @@
+use cosmwasm_std::{Decimal, Decimal256, StdError, StdResult};
+
+/// ## Description
+/// Converts [`Decimal`] to [`Decimal256`].
+pub fn decimal2decimal256(dec_value: Decimal) -> StdResult<Decimal256> {
```
[the remark this answers is not in the document]
Comment (P-Yevhenii) [reply packages/astroport/src/math.rs:5]: Let it be, I don't mind

File: contracts/tokenomics/maker/tests/integration.rs [hunk 3]
```diff
@@ -607,27 +614,44 @@ fn test_maker_collect(
 
         // Check balances
         let amount = Uint128::new(amount);
-        let governance_amount =
```
[the remark this answers is not in the document]
Comment (epanchee) [reply contracts/tokenomics/maker/tests/integration.rs]: This todo still needs to be added

File: contracts/tokenomics/generator_proxy_to_mirror/src/contract.rs [hunk 4]
```diff
@@ -369,21 +370,22 @@ pub fn query(deps: Deps, env: Env, msg: QueryMsg) -> StdResult<Binary> {
...
             )?;
-            let reward_infos = res.reward_infos;
-            let deposit_amount = if !reward_infos.is_empty() {
-                reward_infos[0].bond_amount
-            } else {
-                Uint128::zero()
-            };
-
-            to_binary(&deposit_amount)
+            // let reward_infos = res.reward_infos;
```
Comment (ShadoySV) [inline contracts/tokenomics/generator_proxy_to_mirror/src/contract.rs:380]: Mark this with TODO too

**Review by ShadoySV — APPROVED** [review]

**Review by maksk7 — APPROVED** [review]
Approved, but fix conflicts and contracts compilement

**Pull request merged**